README.md

```diff
@@ -6,14 +6,70 @@
 
 A collection of tools (including data analysis and machine learning applications) that can be used to explore radiation data (temporal and spectral) for external identification of radiation signatures.
 
-## Dependencies
+## Table of Contents
+
+1. [Installation](#Installation)
+
+2. [Usage](#Usage)
+
+## Installation
+
+### Dependencies
 
 * Python 3
 
-Including the following Python packages (found in `requirements.txt`):
+Versions 3.6-3.9 are currently supported by tests. The following Python packages are required to run (found in `requirements.txt`):
 
 * h5py
 * numpy
 * pandas
 * progressbar2
 * scipy
+
+Scripts can be run from the top-level directory:
+
+`python /path/to/RadClass/script`
+
+Or `RadClass` can be installed using pip:
+
+`python -m pip install /path/to/RadClass/. --user`
+
+## Usage
+
+RadClass is designed to be modular, allowing a user to use analysis modules as needed:
+
+<p align="center">
+  <img width="600" src="https://drive.google.com/uc?export=view&id=14RZCOaTmcwKaEjiWHaQeARaxCpx3zAur">
+</p>
+
+An HDF5 data file is specified as input, which is processed by `DataSet`. The user can specify a type of `AnalysisParameters`. For example, `H0` for hypothesis testing, `BackgroundEstimator` for background estimation, etc.
+`RadClass` then uses `DataSet` and the user specified `AnalysisParameters` to run, storing the results for use by the user.
+To see examples of how a user can initialize and run `RadClass`, review /tests/.
+
+`RadClass` expects a data structure as follows:
+
+<p align="center">
+  <img width="600" src="https://drive.google.com/uc?export=view&id=1RQM7Ipy1h4zOBaZlpsVTmIpt_mymwFea">
+</p>
+
+The HDF5 file must have three groups:
+
+1. A vector of epoch timestamps at which the measurement was taken.
+2. If applicable, the live recording time of the detector to correct for dead time. If not applicable, a vector of 1's.
+3. A data matrix of `n` measurements given for `m` bins.
+
+Each groups name must be specified as an input dictionary: `labels`. Integration occurs over the course of the data matrix.
```

Comment:
The sentence beginning with "Integration" is a new paragraph/section on data pre-processing